### ISSUE TITLE
In CFI, colors aren't consistent

### ISSUE THREAD

**Issue by mtomasini** [benflips/nCovForecast#4]
When checking other countries to be showed in the plot for the curve-flattening index, line colors are re-assigned. What would be ideal is that when adding a new country to the plot, the countries that were already there could be of the same color as before.

**Comment by benflips** [comment]
Thanks heaps Matteo. Should have time to look at all this in about 9h.

Excuse the brevity: sent from my phone.

On 15 Mar 2020, at 06:53, Matteo Tomasini <notifications@github.com<mailto:notifications@github.com>> wrote:


When checking other countries to be showed in the plot for the curve-flattening index, line colors are re-assigned. What would be ideal is that when adding a new country to the plot, the countries that were already there could be of the same color as before.

**Comment by mtomasini** [comment]
I think the easiest way would be to attach colors to countries. I will try that now.

**Comment by mtomasini** [comment]
I tried to play around with it. It is quite easy to assign a different color to each country, but with >100 countries, the colors are assigned sequentially to each country, and there isn't enough contrast to make distinctions. I tried to change color palette, and I also played around with the function sample() but the result wasn't satisfying. 

I think the easiest way to deal with it would be to subset the dataset to the first 30 countries or so, but I'm not sure we wanna do that. Are consistent colors more important than a full dataset?
